front/src/components/EliminationsBrackets/EliminationsBrackets.js

```diff
@@ -0,0 +1,97 @@
+import "./EliminationsBrackets.css";
+import EliminationsElement from "./EliminationsElement/EliminationsElement";
+import CompetitorsData32 from "./CompetitorsData32.json";
+import CompetitorsData16 from "./CompetitorsData16.json";
+import CompetitorsData8 from "./CompetitorsData8.json";
+import CompetitorsData4 from "./CompetitorsData4.json";
+import CompetitorsData2 from "./CompetitorsData2.json";
+
+const EliminationsBrackets = () => {
+    const testCompetitorsData = [
+        CompetitorsData32,
+        CompetitorsData16,
+        CompetitorsData8,
+        CompetitorsData4,
+        CompetitorsData2,
+    ];
+
+    const generateOrder = (max) => {
+        let orderArray = [[1], [1, 2]];
+        let len = 2;
+        while (len <= max) {
+            let newOrder = [];
+            for (let i = 0; i < len; i = i + 2) {
+                let sum = len * 2 + 1;
+                let prevArr = orderArray[orderArray.length - 1];
+                newOrder.push(prevArr[i]);
+                newOrder.push(sum - prevArr[i]);
+                newOrder.push(sum - prevArr[i + 1]);
+                newOrder.push(prevArr[i + 1]);
+            }
+            len = len * 2;
+            orderArray.push(newOrder);
+        }
+        return orderArray.reverse();
+    };
+
+    const generatePowerOf2 = (numberOfCompetitors) => {
+        let checkPowerOf2 = Math.log2(numberOfCompetitors);
+        let total = Math.ceil(checkPowerOf2);
+        return Math.pow(2, total);
+    };
+
+    let numberOfPositions = generatePowerOf2(testCompetitorsData[0].length);
+    let orderArray = generateOrder(numberOfPositions);
+
+    const generateRoundColumn = (arrayOfPairs, arrayOfPositions) => {
```

Comment:
Don't describe datatypes in variable names unless it's crucial for the algorithm.
First argument contains data about eliminations by duels (maybe name it eliminationData? duels? don't know) and second should just be positions.